atest/DynamicTypesAnnotationsLibrary.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Union, NewType
+
+from robot.api import logger
+from selenium.webdriver.chrome.webdriver import WebDriver
```

Comment:
This seems to be used only as a forward reference. Is an external dependency really needed for that? Why not use something from the standard library?

Reply:
True, it could be even a dummy class created in test. Will refactor.